frontend/src/pages/Posts.tsx

```diff
@@ -77,6 +79,10 @@ const Posts = () => {
     }
   };
 
+  const handleSearch = () => {
+    fetchPosts(page, 12, searchQuery, filterTags);

```

Comment:
make page size configurable.

Potential solution:
1. Have a drop down on UI to select page size. (if we already have that in UI)
2. Store it environment variable and fetch from there.

Reply:
I haven't mention this in the PR did I ?

Reply:
Let's ignore this for now then.